flow.py

```diff
@@ -109,8 +109,10 @@ def send_packets(self, delay=0.0):
         delay (float): delay until sending packets. Should only be used for
             initial send.
         """
-        while (len(self.unacknowledged_packets) < self.window_size):
-            if (len(self.timed_out_packets) > 0):
+        while len(self.unacknowledged_packets) < self.window_size:
+            # I'm confused because window size doesn't change?
+            self.ns.record_window_size(self.flow_id, self.window_size)
```

Comment:
Window size will change later when we do the congestion stuff. For now we probably shouldn't record it